src/math/average_median.jl

```diff
@@ -11,11 +11,11 @@ median([2, 70, 6, 50, 20, 8, 4])    # returns 8
 median([0])                         # returns 0
 ```
 
-Contributed By:- [Ashwani Rathee](https://github.com/ashwani-rathee)
+Contributed By:- [Ashwani Rathee](https://github.com/ashwani-rathee) and [Rratic](https://github.com/Rratic)
 """
-function median(nums)
+function median(nums::Vector)
```

Comment:
try `function median(nums::Vector{T}) where T <: Number` because `Any` is default in Vector

Reply:
Also, unless there is a conflict with another median method, there is no need to provide types